src/java.base/share/classes/java/lang/foreign/ValueLayout.java

```diff
@@ -649,6 +649,8 @@ public boolean isUnbounded() {
         }
     }
 
+    // Some constants below are accessed from generated bytecode, see BindingSpecializer::emitLoadLayoutConstant
```

Comment:
Which constants "below" ? The comment would have been perfect in the old code, but now it seems the constants have been removed?

Reply:
So, before they were all in a special class (Platform) but now the public layouts are used.

Reply:
What I meant is: the comment you copied referred to the layout definitions which have now been removed. So I think it's better to remove the comment.

Reply:
OK. I thought the comment meant: "Do not change, move or remove any of these constants as they are invisibly used in bytecode generated in BindingSpecializer.". But I can remove the comment.